run.py

```diff
@@ -342,6 +344,17 @@ def get_tier_level(files: str) -> str:
     return "Unknown"
 
 
+def get_html_page(url: str) -> str:
```

Comment:
Would suggest to move this under any helper module.

Reply:
This method is not generic and is specific to this module. I would like to keep it here...

Reply:
ok